src/renderer/components/server-modal-form.jsx

```diff
@@ -232,7 +242,7 @@ export default class ServerModalForm extends Component {
                     name="port"
                     maxLength="5"
                     placeholder="Port"
-                    value={this.state.port}
+                    value={this.state.port || this.state.defaultPort}
```

Comment:
@maxcnunes so i have change here so only if user has not choose any port we gonna apply default one